scripts/scripts.js

```diff
@@ -130,6 +295,31 @@ function findReplaceJSON(jsonObj, data) {
   return jsonObj;
 }
 
+// Custom recursive deep copy function
+function deepCopy(obj) {
```

Comment:
Can probably be replaced with https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

Reply:
I tried to use it it give me error not sure but looks like it is not handling method transfer properly
`function deepCopy(obj) {
  if (typeof obj !== 'object' || obj === null) {
    return obj;
  }

  return structuredClone(obj);
}`
![image](https://github.com/hlxsites/zemax/assets/129520313/fd7d5e17-c563-4441-ae63-cf74cc86c08a)

Reply:
Ah wait, structuredClone does not work with DOM elements. I assumed this are plain JS objects.